### PR TITLE
Sidekiq options support

### PR DESCRIPTION
**Pull request by emq**
Additional work on top of https://github.com/krisleech/wisper-sidekiq/pull/16 according to discussion in that PR.

@krisleech please let me know what do you think - if it's fine with you I will wrap it up! (docs/cleanup according to suggestions)

### CODE REVIEW

File: lib/wisper/sidekiq.rb [hunk 1]
```diff
@@ -20,16 +19,25 @@ def perform(yml)
...
+
+    def broadcast(subscriber, publisher, event, args)
+      options = sidekiq_options(subscriber)
+      Worker.set(options).perform_async(::YAML.dump([subscriber, event, args]))
+    end
+
+    private
+
+    # Listener does not include any module, sidekiq_options are optional thus this ugly check
+    # This is fully backward compatible with current API so it's a fair tradeoff
```
Comment (krisleech) [inline lib/wisper/sidekiq.rb]: I don't think this comment is needed, the code below is self explanatory.

**Comment by krisleech** [comment]
Looks good to me, thanks. I wasn't sure if we need to bother with running the spec on Travis for old versions of Ruby, e.g. 2.2.

There isn't a CHANGELOG file, could you create one please so we can document new features such as this.

**Comment by emq** [comment]
> I wasn't sure if we need to bother with running the spec on Travis for old versions of Ruby, e.g. 2.2.

Won't hurt right? [sidekiq repo for references](https://github.com/mperham/sidekiq/blob/master/.travis.yml#L10-L14)

>  could you create one please so we can document new features such as this

Added, please take a look 👀

**Comment by krisleech** [comment]
:+1: All looks good to me, are you happy?

**Comment by emq** [comment]
If all is fine with you let's :shipit: 🤘

**Pull request merged**